indico/modules/events/contributions/templates/display/_contribution_list.html

```diff
@@ -0,0 +1,57 @@
+{% macro render_displayed_entries_fragment(displayed_num, total_num) -%}
+    <span id="filtering-state" class="i-button label icon-list"
+          title="{% trans %}Displaying {{displayed_num}} out of {{total_num}} contributions{% endtrans %}">
+        <strong>{{ displayed_num }}</strong> / {{ total_num }}
+    </span>
+{%- endmacro %}
+
+{% macro render_contribution_list(event, contributions) %}
+    {% for contrib in contributions -%}
+        <div class="contribution-row" data-friendly-id="{{ contrib.friendly_id }}">
+            <div class="contrib-title info" data-searchable="{{ contrib.title }}">
+                <span class="value">
+                    <a href="{{ url_for('contributions.display_contribution', contrib) }}">{{ contrib.title }}</a>
+                </span>
+            </div>
+            {% if contrib.description -%}
+                <div class="info">
+                    <span class="label">{% trans %}Description{% endtrans %}</span>
+                    <span class="value">{{ contrib.description|strip_tags }}</span>
+                </div>
+            {%- endif %}
+            {% if contrib.track -%}
+                <div class="info" data-searchable="{{ contrib.track.title }}">
+                    <span class="label">{% trans %}Track:{% endtrans %}</span>
+                    <span class="value">{{ contrib.track.title }}</span>
+                </div>
+            {%- endif %}
+            {% if contrib.type -%}
+                <div class="info" data-searchable="{{ contrib.type.name }}">
+                    <span class="label">{% trans %}Type:{% endtrans %}</span>
+                    <span class="value">{{ contrib.type.name }}</span>
+                </div>
+            {%- endif %}
+            {% if contrib.session -%}
+                <div class="info" data-searchable="{{ contrib.session.title }}">
+                    <span class="label">{% trans %}Session:{% endtrans %}</span>
+                    <span class="value">
+                        <a href="{{ url_for('sessions.display_session', contrib.session) }}">{{ contrib.session.title }}</a>
+                        <span class="session-bg-color" style="background: #{{ contrib.session.colors.background }};"></span>
+                    </span>
+                </div>
+            {%- endif %}
+            {% if contrib.speakers -%}
+                <div class="info contrib-speakers">
+                    <span class="label">{% trans %}Presented by{% endtrans %}</span>
+                    <span class="value">
+                        <strong>{{- contrib.speakers|map(attribute='full_name')|join(', ') -}}</strong>
+                        {% if contrib.timetable_entry -%}
+                            on <strong>{{ contrib.timetable_entry.start_dt|format_date }}</strong>
```

Comment:
Missing translation. See [this](https://github.com/indico/indico/blob/devel/indico/modules/events/templates/display/common/_legacy.html#L8) for a similar situation.